src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -5,6 +5,9 @@ var f = require('{0}'),
     util = require('util');
 
 return function (context, callback) {{
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
+    setImmediate(() => {{}});
```

Comment:
I also vote to remove the old hack. If anyone was depending on that (which I seriously doubt) -- then they'd have a script that didn't work in 'pure' node but did work in Functions. That seems bad and we should avoid that.

---
In reply to: [78418906](https://github.com/Azure/azure-webjobs-sdk-script/pull/664#discussion_r78418906) [](ancestors = 78418906)